website/ibm.erb

```diff
@@ -241,6 +244,27 @@
           <li<%= sidebar_current("docs-ibm-resource-infra") %>>
             <a href="#">Infrastructure Resources</a>
             <ul class="nav nav-visible">
+              <li<%= sidebar_current("docs-ibm-resource-cis") %>>
```

Comment:
Why compute_autoscale_group it should be the name of the corresponding file (All entries need to be changed)

Reply:
Result of cut and paste. resolved.